### PR TITLE
allowed permission for mailbox on role of ithead

### PR DESCRIPTION
**Pull request by pravinpimpale**

### CODE REVIEW

File: app/Http/Controllers/MailboxesController.php [hunk 1]
```diff
@@ -137,7 +137,7 @@ public function update($id)
 
         $user = auth()->user();
         $mailbox_user = $user->mailboxesWithSettings()->where('mailbox_id', $id)->first();
-        if (!$mailbox_user && $user->isAdmin() || $user->isITHead()) {
+        if (!$mailbox_user && $user->isAdmin() || !$mailbox_user &&$user->isITHead()) {
```
Comment (umar-ashrfy) [inline app/Http/Controllers/MailboxesController.php]: Change to
if (!$mailbox_user && ($user->isAdmin() || $user->isITHead())) {

File: Modules/Crm/Http/routes.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-Route::group(['middleware' => ['web', 'auth', 'roles'], 'roles' => ['user', 'admin'], 'prefix' => \Helper::getSubdirectory(), 'namespace' => 'Modules\Crm\Http\Controllers'], function()
+Route::group(['middleware' => ['web', 'auth', 'roles'], 'roles' => ['user', 'admin','ithead'], 'prefix' => \Helper::getSubdirectory(), 'namespace' => 'Modules\Crm\Http\Controllers'], function()
```
Comment (umar-ashrfy) [inline Modules/Crm/Http/routes.php]: Please add other roles as well

**Review by umar-ashrfy — CHANGES_REQUESTED** [review]

File: Modules/Crm/Http/routes.php (hunk 2 above)
Comment (pravinpimpale) [reply Modules/Crm/Http/routes.php]: added other roles as well

File: app/Http/Controllers/MailboxesController.php (hunk 1 above)
Comment (pravinpimpale) [reply app/Http/Controllers/MailboxesController.php]: It is done

**Pull request merged**